src/helper/Redux/types/reminder.types.js

```diff
@@ -1,5 +1,16 @@
 // TODO: types of reminder
 export const FETCH_REMINDER = 'FETCH_REMINDER';
+export const FETCH_REMINDER_SUCCESS = 'FETCH_REMINDER_SUCCESS';
+export const FETCH_REMINDER_FAILURE = 'FETCH_REMINDER_FAILURE';
+
```

Comment:
also, review this status indicator I created. and trigger the value like `false` to `true` right? using reducers.